tests/test_routes.py

```diff
@@ -310,4 +310,88 @@ def test_delete_item_item_not_found(self):
         """ Delete an Item where item does not exist"""
         order = self._create_orders(1)[0]
         resp = self.app.delete("/orders/{}/items/{}".format(order.id, 0))
-        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
\ No newline at end of file
+        self.assertEqual(resp.status_code, status.HTTP_404_NOT_FOUND)
+
+    def test_update_order_item(self):
+        """ Update an existing Order's Item """
```

Comment:
the docstrings for all the test functions are the same. Could we please change them? Thanks